app/exporters/shopify/product_exporter.rb

```diff
@@ -0,0 +1,90 @@
+module Shopify
+  class ProductExporter
```

Comment:
This class does too much.

It should be seperated in multiple different classes. I had worked on refactoring this and something like that would be cleaner:

https://github.com/glossier/solidus_retail/blob/8bf611f0ee385d556958ff2ea2dd656fbf924544/app/exporters/shopify/product_exporter.rb